src/renderer/store/global.ts

```
@@ -0,0 +1,16 @@
+import { ref, computed } from "vue";
```

ESLint

[error] 'computed' is defined but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)